app/assets/javascripts/roster/sorts/fluency_sort.js

```diff
@@ -0,0 +1,25 @@
+// Extend TableSort with Fluency levels
+(function(){
+
+  function clean_cell(i) {

```

Comment:
Could you remove the `clean_cell`, it looks like it's not doing anything other than calling `$.trim` in each case?